src/schema/v2/partner.ts

```diff
@@ -39,44 +44,89 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
   name: "Partner",
   interfaces: [NodeInterface],
   fields: () => {
+    // This avoids a circular require
+    const ArtistPartnerConnection = connectionWithCursorInfo({
+      name: "ArtistPartner",
+      nodeType: ArtistType,
+      edgeFields: partnerArtistFields,
+    }).connectionType
+
     return {
       ...SlugAndInternalIDFields,
       cached,
+      artistsConnection: {
+        description: "A connection of artists at a partner.",
+        type: ArtistPartnerConnection,
+        args: pageable({
+          sort: ArtistSorts,
+          representedBy: {
+            type: GraphQLBoolean,
+          },
+        }),
+        resolve: ({ id }, args, { partnerArtistsForPartnerLoader }) => {
+          const pageOptions = convertConnectionArgsToGravityArgs(args)
+          const { page, size, offset } = pageOptions
+
+          interface GravityArgs {
+            page: number
+            size: number
+            total_count: boolean
+            sort: string
+            represented_by: boolean
+          }
+
+          const gravityArgs: GravityArgs = {
+            total_count: true,
+            page,
+            size,
+            sort: args.sort,
+            represented_by: args.representedBy,
+          }
+
+          return partnerArtistsForPartnerLoader(id, gravityArgs).then(
+            ({ body, headers }) => {
+              return connectionFromArraySlice(body, args, {
+                arrayLength: parseInt(headers["x-total-count"] || "0", 10),
+                sliceStart: offset,
+                resolveNode: node => node.artist,
+              })
+            }
+          )
+        },
+      },
       artworksConnection: {
         description: "A connection of artworks from a Partner.",
         type: artworkConnection.connectionType,
         args: pageable(artworksArgs),
-        resolve: ({ id }, options, { partnerArtworksLoader }) => {
+        resolve: ({ id }, args, { partnerArtworksLoader }) => {
           const { page, size, offset } = convertConnectionArgsToGravityArgs(
-            options
+            args
           )
 
           interface GravityArgs {
             exclude_ids?: string[]
             page: number
-            published: boolean
```

Comment:
Why the removal of `published` here?

Reply:
The corresponding [loader](https://github.com/artsy/metaphysics/blob/ca2f97377170399ff86d3a91c00c284733c66979/src/lib/loaders/loaders_without_authentication/gravity.ts#L43) , winds up calling [this endpoint](https://github.com/artsy/gravity/blob/367f083c7dda7f7ab7828d6708e930e3f8b0c5f9/app/api/v1/partners_artworks_endpoint.rb#L81) in Gravity.

That endpoint should be receiving a `published: true` param, should undo this change.

But the [below comment](https://github.com/artsy/metaphysics/pull/2014/files?file-filters%5B%5D=.js&file-filters%5B%5D=.ts#r338141149) still applies, for that field, `published` doesn't make sense as a gravity arg.

Reply:
Ok, this should be fixed.